test/test_to_be_named.jl

```diff
@@ -0,0 +1,16 @@
+# example of to_be_named
+
+using Distributions
+
+include("../src/WRS.jl")
+using .WRS
+
+n = 10000
+d = Normal()
+
+# generate data
+sample_a = rand(d, n)
+sample_b = rand(d, n)
+
+# run comparison
+results = to_be_named(sample_a, sample_b, hd)
```

Comment:
The more I think about the API here, the more I believe there is probably some elegant API design based on broadcasting over quantiles... But lets merge this for now and reconsider later, if anyone has the cycles for it.

Reply:
What should I name the function?